scripts/demoScripts/reportsMockScript.js

```diff
@@ -0,0 +1,175 @@
+const Client = require('node-rest-client').Client;
+const { createMoodEntries, createMeditationEntries, createMedicationAdherence } =  require('./createRecordsScript');
+
+//URL's
+const authUrl = `${process.argv[2]}/auth/login`;  //"http://localhost:4000/api/v1/auth/login"
+const patientsUrl = `${process.argv[3]}/patients`;   //"http://localhost:5000/v1/patients"
+const clientSecret = process.argv[4];
+
+// Email and password of the patient
+const email = process.argv[5].replace(/^"(.*)"$/, '$1'); 
+const password = process.argv[6].replace(/^"(.*)"$/, '$1');
+
+// TODO: JCB once this script creates medications we will need to do the same regex replace on these values
+// Email and password of the clinician
+const email2 = process.argv[7];
+const password2 = process.argv[8]; 
+
+// Init value determines if we are creating medications for each user or not
+const init = (process.argv[9] ? process.argv[9]: false);
+
+var authToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NywidXNlcm5hbWUiOiJSdWJ5ZUBhbWlkYS5jb20iLCJlbWFpbCI6IlJ1YnllQGFtaWRhLmNvbSIsInNjb3BlcyI6WyIiXSwiaWF0IjoxNTIwMjY5OTA5LCJleHAiOjE1MjAyNzY1MDl9.lBsjsQtHxuI8E5C7VHaxkulZZugkbk0FFYl_rT580Bo';
```

Comment:
`authTokenAdmin` might be clearer. Have to carefully read functions to distinguish meaning of existing `authToken1` and `authToken2`